dips/dip-7.md

```diff
@@ -14,40 +14,56 @@ This DIP outlines the process by which technical working groups (TWG’s) are fo
+- The group’s scope and goal
+- The limits of its focus
+- The expected final output(s), typically as a DIP, report, or recommendation
+- Expected duration of operation (typically 3-6 months)
+- The initial facilitator (up to two individuals)
+- A brief description of ideal desired member profiles so other community or Association members can join
+- Whether the TWG is intended to be public or by invitation (public is default)
 
-Once the proposal is created, the facilitator will circulate it in public slack channels relevant to the topic. The facilitator is also encouraged to proactively share the proposal with Association members that are likely to be impacted by this TWG or that could contribute to it. The facilitator must circulate the proposal over 1-2 weeks for sufficient interest to gather.
+Once the proposal is created, the facilitator will circulate it in public slack channels and other mediums relevant to the topic. The facilitator is also encouraged to proactively share the proposal with Association members that are likely to be impacted by this TWG or that could contribute to it. The facilitator must circulate the proposal over 1-2 weeks for sufficient interest to gather.
 
-After gathering interest, the facilitator must choose members to be part of the TWG, add them to the proposal, and identify their roles (co-facilitator, contributor, reviewer, etc.). The proposal must have representation from a minimum of 3 Association members to build collaboration and as a tool to determine relevance to the entire association, rather than an individual member.
+Anyone can join a TWG by asking for the facilitator(s) to add them to the proposal (and at any time afterwards) and identifying their roles (co-facilitator, contributor, reviewer, etc.). It is recommended for the proposal to have representation from a minimum of 3 Association members to build collaboration and as a tool to determine relevance to the entire association, rather than an individual member.
 
-The facilitator is responsible for promoting and explaining the proposed TWG to Diem members and must lead it through the voting process by the Technical Steering Committee (TSC). They are also responsible for recruiting other members to the TWG as needed on an ongoing basis, and must help find a replacement facilitator if they must cease work on the initiative.
+The facilitator is responsible for promoting and explaining the proposed TWG to Diem members and the community, and must lead it through the voting process by the Technical Steering Committee (TSC). They are also responsible for recruiting other members to the TWG as needed on an ongoing basis, and must help find a replacement facilitator if they must cease work on the initiative.
 
 Once the TWG proposal is presented to the TSC, a simple majority vote in favor forms the TWG. The TWG can then proceed in its stated work and make decisions as needed in the areas covered by its focus.
 
```

Comment:
We need a filtering mechanism of what gets percolated to the TSC, as its time is limited. Again, I am not thinking about Association members initiating a TWG, but the public.

The Lead Maintainer determines the agenda for discussions with the TSC, and also what merits being a DIP. It is most natural to say that the Lead Maintainer determines if a TWG proposal merits being brought in front of the TSC. 

Note that people are welcome to "lobby" with Maintainers, with the CTO, etc., and it would be unthinkable that the LM will ignore it.